src/muya/lib/parser/render/index.js

```diff
@@ -13,6 +13,7 @@ class StateRender {
   constructor (muya) {
     this.muya = muya
     this.eventCenter = muya.eventCenter
+    this.codeCache = new Map()
     this.loadImageMap = new Map()
     this.loadMathMap = new Map()
     this.mermaidCache = new Set()
```

Comment:
How are these caches cleared/invalidated, may this lead to memory leaks?

Reply:
This can be clear after each `render` and `partialRender`.